WorkAndFun/src/main/resources/templates/announcementDetails.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/html">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!--/* Bootstrap CSS */-->
+    <link th:href="@{/css/bootstrap.min.css}" rel="stylesheet"/>
+
+    <link th:href="@{/css/style.css}" rel="stylesheet"/>
+
+    <title>Szczegóły ogłoszenia</title>
+</head>
+
+
+<body class="bg-light p-3">
+<div class="flex">
+    <div class="container-lg">
+        <div class="row mb-1">
+            <div class="col-md-12" th:insert="~{fragments/navbar :: nav}">
+            </div>
+        </div>
+        <div class="row">
+            <div class="m-2 col-lg-6 bg-default">
```

Comment:
chyba musisz zamienić ten bg-default na coś innego, bo najechaniu myszką trochę to smutnie wygląda, zwłaszcza że header i opis mają inny background color wtedy

Reply:
dodałem nowe style css dla ramek announcement-details i spiąłem, starałem się zrobić lekki wygląd, już tak nie smuci :)